Libadwaita-Dark/styles/sidebar.less

```diff
@@ -49,6 +49,10 @@
 	margin-bottom: 8px;
 	height: 36px;
 
+	.text {
+		height: unset !important;
+	}
```

Comment:
Should add `min-height: 36px;` to ensure sidebar items do not become more compact